apps/emqx_bridge/test/emqx_bridge_mqtt_SUITE.erl

```diff
@@ -551,7 +552,7 @@ t_mqtt_conn_bridge_egress_reconnect(_) ->
         #{
             <<"status">> := Status,
             <<"metrics">> := #{
-                <<"matched">> := 3, <<"success">> := 1, <<"failed">> := 0, <<"queuing">> := 2
+                <<"matched">> := 1, <<"success">> := 1, <<"failed">> := 0, <<"queuing">> := 2
```

Comment:
It should be 3 messages matched to to bridge, 1 success and 2 queued (because it is disconnected).

Reply:
Since the `matched` metric increases [here](https://github.com/emqx/emqx/pull/9128/files#diff-ddb2b9013a33d2ea0296b26a945afc4d32e3b346e5217223a8ee4a31d4ffbce2R424) after a successful ETS lookup, it seems that the `matched` count in this test is now `1`, because its status is no longer `connected`.  

You mean we should also increase `matched` when the resource status is `stopped | connecting | disconnected`?

Reply:
Yes, I think in case of disconnected, the message is handled by the bridge (matched), but the bridge doesn't send it, instead it is queued in the bridge buffer.
> You mean we should also increase matched when the resource status is stopped | connecting | disconnected?

Reply:
:heavy_check_mark: